src/index.test.ts

```diff
@@ -0,0 +1,27 @@
+import request from 'supertest';
+
+import app from './index';
```

Comment:
Here you import an app that then tries to connect to the database. It is not very convenient to access the database within unit testing. It is better to mock the DB responses where possible and not to test where it is impossible. Coverage does not have to be an end in itself.